### ISSUE TITLE
Drop in accuracy after rewinding convnext training 

### ISSUE THREAD

**Issue by AlaaKhaddaj** [mlfoundations/open_clip#487]
Hello,

I am trying to replicate the improvement you got in convnext using rewinding. However, I am noticing a big drop in accuracy. Can you please provide more details about that? 

Thanks!

**Comment by AlaaKhaddaj** [comment]
Yes, we are using a global batch size of ~70k. We are not sure exactly how you are doing the rewinding? Are you starting the training from the last checkpoint, with a larger LR and batch size? How many epochs are you rewinding?

**Comment by rwightman** [comment]
@AlaaKhaddaj it is not retraining for 256 checkpoint epochs ('virtual' epochs as they are not full dataset passes), only a 10% rewind which in this case means I resumed from checkpoint 230, so it reran the last 16. Basically setting the initial LR to 2e-3 instead of 1e-3 (so the resume LR will be based on 230/256 schedule wise in the cosine and slightly altered by the # steps from the global batch size change). I'd say the bump in global batch is a bit of a factor, from 82->95k. Augmentation was increased.

The main gain though was from averaging the two (original and rewind) as per the model soup idea. That is also why I varied the augmentations slightly between the two as that seems to improve the averaged result.
